scargo/commands/check.py

```diff
@@ -186,15 +186,25 @@ def check(self) -> None:
         super().check()
 
     def check_file(self, file_path: Path) -> CheckResult:
+        copyrights = self.copyright_desc.split("\n")[:-1]
```

Comment:
I think `copyright_lines` would be a better name for this variable.

Reply:
done